serverless/src/shared/users/index.js

```diff
@@ -228,6 +228,26 @@ const createLoginCode = async ({ email, userId, userPoolIdOverwrite }) => {
   return secretLoginCode;
 };
 
+const getAllCognitoUsers = async (
+  cognitoUsers = [],
+  paginationToken = null
```

Comment:
What is this paginationToken used for?

Reply:
The AWS services (Cognito but dynamo as well) work with pagination, meaning: to get all users I wrote a recursive function which makes multiple requests (always starting on the next "page") and its users to an array each time.

Reply:
Cool, thanks! :)